roles/ceph-common/tasks/installs/install_rh_storage_on_debian.yml

```diff
@@ -87,6 +87,6 @@
 
 - name: install red hat storage nfs gateway
   apt:
-    name: nfs-ganesha-fsal
+    name: nfs-ganesha-ceph
```

Comment:
Note: this will break on RH Ceph Storage 2 where the nfs-ganesha 2.4.5 sub-package is just named `nfs-ganesha-fsal` (matching the nfs-ganesha 2.4.5 upstream debian package)

Reply:
Which "upstream Debian" package are you referring to? I ask because https://packages.debian.org/search?keywords=nfs-ganesha-fsal&searchon=names&suite=stable&section=all turns up nothing.

Reply:
Yeah, the official Debian/Ubuntu ones are not well-maintained.

Here are the upstream (2.4.5) ones upon which RH Ceph Storage 2 was based: https://launchpad.net/~gluster/+archive/ubuntu/nfs-ganesha/+packages

Reply:
Yes, *these packages do not contain the Ceph FSAL.* That is exactly why I am sending this PR.

Reply:
The point I'm getting at is that `install_rh_storage_on_debian.yml` does not use any PPA; it's installing the downstream RHCS 2 product. For better or worse, the `stable-2.2` branch needs to support RH Ceph Storage 2.

Reply:
Oh, so you're saying in RHCS2, on Debian, you're not using any upstream binary packages but you're _rebuilding_ the ones from upstream, with the Ceph FSAL enabled? Fair enough, then the only thing that needs to happen is for me to roll back the change to `install_rh_storage_on_debian.yml` to where it continues to install `nfs-ganesha-fsal`, and the rest of the PR can remain as-is — correct?

Reply:
@fghaas you've got it now, please accept my apologies for not making this clear earlier.

Have all of these changes gone into master first?

Reply:
In master, I have no idea what the plan for nfs-ganesha is that doesn't break on Jewel or Kraken (which master ostensibly supports) — see #1907. So this PR just unbreaks something that is broken specifically in `stable-2.2`. In `master`, I don't know how to properly unbreak things.